src/components/VirtualizedListView/index.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { _cs } from '@togglecorp/fujs';
+
+import ListView, { Props as ListViewProps } from '../ListView';
+import {
+    OptionKey,
+    GroupCommonProps,
+} from '../List';
+
+import styles from './styles.css';
+
+export type Props<D, P, K extends OptionKey> = {
+    itemHeight: number;
+    elementRef?: React.RefObject<HTMLDivElement>;
+    buffer?: number;
+} & ListViewProps<D, P, K, GroupCommonProps, OptionKey>
+
+function VirtualizedListView<D, P, K extends OptionKey>(props: Props<D, P, K>) {
+    const {
+        className,
+        itemHeight = 16,
+        elementRef: elementRefFromProps,
+        data = [],
+        buffer = 1,
+        pending,
+        ...otherProps
+    } = props;
+
+    const [scrollOffset, setScrollOffset] = React.useState<number>(0);
```

Comment:
Can we get the default value for this from props? It might be useful if we need to jump to a certain element on render. Even better would be to have a prop that takes in key/index of the item to show initially.